chapters/annotations.tex

```diff
@@ -268,6 +268,12 @@ \section{Annotations for Simulation Experiments}\doublelabel{annotations-for-sim
 relative integration tolerance (Tolerance) for simulation experiments to
 be carried out with the model or block at hand. If StartTime is not specified it is assumed to be \lstinline!0.0!.
 
+The experiment annotation can be inherited, the local experiment (if existing) only override the specific options in the normal way for modifiers.
+\begin{nonnormative}
+This implies that it may be useful to construct a partial model (e.g., a template for a number of similar test cases)
+containing an experiment annotation, since the experiment applies to models inherting from this base class.
```

Comment:
I see it more as the main reason behind, rather than an implication:
```suggestion
The inheritance makes it useful to have an \lstinline!experiment!-annotation also in partial models, e.g., a template for a number of similar test cases.
```